extensions/amp-video-docking/0.1/controls.js

```diff
@@ -395,7 +456,9 @@ export class Controls {
    * @param {number} height
    */
   positionOnVsync(scale, x, y, width, height) {
-    const {container, dismissContainer_: dismissContainer} = this;
+    this.area_ = layoutRectLtwh(x, y, width * scale, height * scale);
```

Comment:
`mousemove` evaluates coordinates based on bounding box, so when the docked box changes, controls would toggle intermittently while cursor was still in visible area. Updating the local `area_` box fixes this.